sys/src/cmd/usb/usbd/usbd.c

```diff
@@ -182,11 +182,11 @@ newhub(char *fn, Dev *d)
 	}
 	devctl(h->dev, "hub");
 	ud = h->dev->usb;
-	if(h->isroot)
-		devctl(h->dev, "info roothub csp %#08ux ports %d",
+	if (h->isroot) {
+		devctl(h->dev, "info roothub csp %#08x ports %d",
 			0x000009, h->nport);
-	else{
-		devctl(h->dev, "info hub csp %#08ulx ports %d %q %q",
+	} else {
+		devctl(h->dev, "info hub csp %#08x ports %d %q %q",
```

Comment:
Oh god, seems Ron and me forgot these two :-D hahahaha. Believe me, it was a pain of two weeks moving to ansi standard for this.